jsonnetfile.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": 1,
+    "dependencies": [
+        {
+            "source": {
+                "git": {
+                    "remote": "https://github.com/bitnami-labs/kube-libsonnet"
+                }
+            },
+            "version": "v1.14.6"
+        }
+    ],
+    "legacyImports": true
+}
```

Comment:
Please remove this file from git

Reply:
but then `make compile` doesn't work at all anymore. I would have to remove the make targets as well. I thought I could align this component a little bit more to component testing like espejo

Reply:
You just need to run `jb install`, right?

Reply:
Ah sorry, my bad. I thought it was the .lock file. all good 👍